server/service/src/sync/translations/report.rs

```diff
@@ -121,6 +121,9 @@ impl SyncTranslation for ReportTranslation {
             comment,
             sub_context,
             argument_schema_id,
+            is_custom: todo!(),
+            version: todo!(),
+            code: todo!(),
```

Comment:
```suggestion
            is_custom: true,
            version: 1.0.to_string(),
            code: id,
```
I don't think it's a good idea to check in code to develop with todo!() like this as it breaks tests which could cause people to spend time trying to fix, and/or we might ignore for ever.

Not a major, `todo!` is much better than putting something wrong it at least!
But I think we know what to put in now, so lets do that now... IMHO